### PR TITLE
first 'pom pattern' implementation, done

### PR DESCRIPTION
**Pull request by JuanImbachi**
image
image

### CODE REVIEW

**Comment by holgiosalos** [comment]
Hey Juan, congrats for reaching out to this step! I will be busy this night and tomorrow, so I apologize in advance for the delay on this review. 

It will take me a while to review it. Please, make sure that the shipping address includes the acceptance of the Terms of Service and also the proceed to checkout action.

File: src/page/index.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+export { MenuContentPage } from './menu-content.page';
+export { ProductAddedModalPage } from './product-added-modal.page';
+export { AddressStep } from './address-step.page';
```
Comment (yesid-lopez) [inline src/page/index.ts:3]: add page at the end. For example: `export { AddressStepPage } from './address-step.page';`

File: src/page/index.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+export { BankPaymentPage } from './bank-payment.page';
+export { OrderSummaryPage } from './order-summary.page';
+export { PaymentStep } from './payment-step.page';
```
Comment (yesid-lopez) [inline src/page/index.ts:6]: same

File: src/page/index.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+export { ProductListPage } from './product-list.page';
+export { ShippingStep } from './shipping-step.page';
```
Comment (yesid-lopez) [inline src/page/index.ts:8]: same

File: src/page/index.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+export { SignInStep } from './sign-in-step.page';
```
Comment (yesid-lopez) [inline src/page/index.ts:9]: same

File: src/page/index.ts [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+export { SummaryStep } from './summary-step.page';
```
Comment (yesid-lopez) [inline src/page/index.ts:10]: same

**Review by yesid-lopez — APPROVED** [review]

**Pull request merged**